minor fixes (#1613)
greenbone/openvas-scanner: rust/openvasd/README.md

| TLS Client Certificates  | --tls-client-certs      |               | tls                                | client_certs      | TLS_CLIENT_CERTS         | Path to client TLS certs enables mTLS                                                                                                                                     |                               |
| Enable get scans         | --enable-get-scans      |               | endpoints                          | enable_get_scans  | ENABLE_GET_SCANS         | Enables GET /scans endpoint                                                                                                                                               | false                         |
| API key                  | --api-key               |               | endpoints                          | key               | API_KEY                  | API key that must be set as X-API-KEY header to gain access. If none is given, api-key authorization is disabled                                                          |                               |

> | Wrapper Type             | --scanner-type          |               | scanner                            | type              | WRAPPER_TYPE             | Type of wrapper used to manage scans, currently only `OSPD` is available                                                                                                  | OSPD                          |

Reviewer: ```suggestion
| Scanner Type             | --scanner-type          |               | scanner                            | type              | SCANNER_TYPE             | Type of wrapper used to manage scans, currently `openvas` and `ospd` are available                                                                                                  | ospd                          |
```

Reviewer: missing option and fix EnvVar

Reviewer: Also missing  the new `REDIS_URL` environment variable in the options table.
